build/dockerfiles/Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
 
 ARG MODULE_PATH
 ARG EXTENSION_ZIP_ADDRS
+RUN mkdir -p $GOPATH/src/extension
```

Comment:
why extension dir need be located within GOPATH?

Reply:
I remove extension dir to /tmp/dicehub-extension now